scripts/cloudfoundry/init/env.properties

```diff
@@ -1,17 +1,17 @@
 MAVEN_REMOTE_REPOSITORIES_REPO1_URL=https://repo.spring.io/libs-snapshot
-SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_ORG=scdf
-SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SPACE=bash-integration
+SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_ORG=stream
+SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SPACE=metrics
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_API_TIMEOUT=300
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_URL=
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SKIP_SSL_VALIDATION=true
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_USERNAME=
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_PASSWORD=
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES=rabbit
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_DOMAIN=
-SPRING_CLOUD_DATAFLOW_SERVER_DOWNLOAD_URL=http://repo.spring.io/snapshot/org/springframework/cloud/spring-cloud-dataflow-server-cloudfoundry/1.2.0.BUILD-SNAPSHOT/spring-cloud-dataflow-server-cloudfoundry-1.2.0.BUILD-SNAPSHOT.jar
+SPRING_CLOUD_DATAFLOW_SERVER_DOWNLOAD_URL=http://repo.spring.io/libs-snapshot/org/springframework/cloud/spring-cloud-dataflow-server-cloudfoundry/1.1.1.BUILD-SNAPSHOT/spring-cloud-dataflow-server-cloudfoundry-1.1.1.BUILD-20170215.152655-32.jar
```

Comment:
I'll bump this back to 1.2.x release. This file is actually a placeholder that users of the tests are expected to override most entries.